File: Week4/filehand.py
# Writing to a file in normal mode
with open('data.txt', 'w') as file:
    file.write('Hello, this is a text file.\n')

# Reading from a file in normal mode
with open('data.txt', 'r') as file:
    data = file.read()
    print("Data read from file in normal mode:")
    print(data)

# Writing to a file in binary mode
with open('data.bin', 'wb') as file:
    file.write(b'Binary data goes here.')

# Reading from a file in binary mode
with open('data.bin', 'rb') as file:
    data = file.read()
    print("\nData read from file in binary mode:")
    print(data)

File: Week4/filehand.m
% Writing to a file in normal mode
fileID = fopen('data.txt', 'w');
fprintf(fileID, 'Hello, this is a text file.\n');
fclose(fileID);

% Reading from a file in normal mode
fileID = fopen('data.txt', 'r');
data = fread(fileID, '*char');
fclose(fileID);
disp("Data read from file in normal mode:")
disp(data)

% Writing to a file in binary mode
fileID = fopen('data.bin', 'wb');
fwrite(fileID, 'Binary data goes here.');
fclose(fileID);

% Reading from a file in binary mode
fileID = fopen('data.bin', 'rb');
data = fread(fileID, '*char');
fclose(fileID);
disp("Data read from file in binary mode:")
disp(data)
